Fitbit.Portable.Tests/FatTests.cs

```diff
@@ -16,7 +16,7 @@ public class FatTests
     {
         public Fixture fixture { get; set; }
 
-        [TestFixtureSetUp]
+        [OneTimeSetUp]
```

Comment:
New one on me :)

Reply:
Our CI build agents didn't like `TestFixtureSetUp` used with V3 of the test framework ;-|